### DISCUSSION TITLE
Thank you for reviving this! I managed to "make it work" with my dock too, even if it did not work with it.

### DISCUSSION THREAD

**Discussion by weltall2** [kangyu-california/PersistentWindows#173] [General]
I can't believe I am using Persistent Windows after so many years. In the past I used this cause my PC had an issue that had nothing with the DisplayPort. Now that I got a DisplayPort monitor myself, I ended up in need of it for it's original cause.

The only thing it did not work with is my ancient ObjectDock dock, that it would remain on the first monitor. I saw that it was mentioned in the description that for support we should check events 9990 and 9999. I just added a task when event 9990 is triggered, to restart ObjectDock, with a delay of 60, so it won't trigger all the times PersistentWindows triggers when monitor wakes up. Now finally everything goes back to it's place!

Thank you one more for keeping this alive! Cheers mate!

**Comment by kangyu-california** [comment]
Will investigate into the compatibility issue with ObjectDocker when I got time.
Thanks for the feedback!
